src/main/java/Duke.java

```diff
@@ -1,27 +1,86 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
 import java.util.Scanner;
 import java.util.ArrayList;
 import java.util.List;
 
 public class Duke {
+    private static List<Task> store = new ArrayList<>();
 
+    private static void readSaveFile(String filePath) throws FileNotFoundException, EmptyDescriptionException {
```

Comment:
I like that the methods are private since they are only used within the class itself 😄👍